myselfのコピー.css

```diff
@@ -73,8 +75,11 @@
 }
 
 .main{
-    height:230vh;
-    background-color:#FDF5E6;;
+  
+    background-color:#FDF5E6;
+}
+.main ol,h3{
+    color:sienna;
```

Comment:
better to be consistent, and use one type of color system (hex or rgba or just names)